crashsimilarity/downloader.py

```diff
@@ -0,0 +1,76 @@
+from datetime import timedelta
```

Comment:
So, could you build a Socorro, Bugzilla and Cache module? The Socorro and Bugzilla modules will use the Cache module.